lms/templates/instructor/instructor_dashboard_2/membership.html

```diff
@@ -37,7 +37,15 @@
       ${_("You will not get notification for emails that bounce, so please double-check spelling.")} </label>
     <textarea rows="6" name="student-ids" placeholder="${_("Email Addresses/Usernames")}" spellcheck="false"></textarea>
   </p>
-
+  <input type="hidden" id="is_course_white_label" value="${section_data['is_white_label']}">
+  % if section_data['is_white_label']:
+  <p>
+    <label for="reason-field-id">
+      ${_("Enter the reason why the students are Enroll or unEnroll manually.")}
+      ${_("The reason field cannot be left blank.")} </label>
```

Comment:
Can I suggest the following text:

"Enter the reason why the students are to be manually enrolled or unenrolled. This cannot be left blank and will be recorded and presented in Enrollment Reports. Therefore, please given enough detail to account for this action."